app/ldap/server/sync.js

```diff
@@ -201,6 +336,30 @@ export function syncUserData(user, ldapUser) {
 		}
 	}
 
+	if (settings.get('LDAP_Sync_User_Data_Groups') === true) {
+		_.each(userRoles, function(roleName) {
+			const add = Roles.addUserRoles(user._id, roleName);
+			if (settings.get('UI_DisplayRoles') && add) {
+				Notifications.notifyLogged('roles-change', {
+					type: 'removed',
```

Comment:
wrong type.

Reply:
What should it be? I copied the "removed" type from here: https://github.com/RocketChat/Rocket.Chat/blob/3db08c7673b8b626906cd714209c8d28c6e66847/app/authorization/server/methods/removeUserFromRole.js#L59